packages/console/src/components/UserRolesTransfer/components/TargetRoleItem/index.module.scss

```diff
@@ -15,6 +15,8 @@
     flex: 1 1 0;
     display: flex;
     align-items: center;
+    margin-right: _.unit(2);
+    overflow: hidden;
```

Comment:
since you have the name text-overflow ellipsis set. no need to set parent wrapper overflow hidden?